appcircle_reactnative_test_report/1.0.4/component.yaml

```diff
@@ -0,0 +1,41 @@
+platform: Common
+buildPlatform: ReactNative
+displayName: Test Reports
+description: Parse test and coverage results.
+docsUrl: https://docs.appcircle.io/continuous-testing/react-native-testing/react-native-unit-testing#generating-test-report
+requiredComponents:
+precedingComponents: "appcircle_reactnative_unit_test,appcircle_reactnative_ui_test"
+followingComponents: "appcircle_export_build_artifacts"
+webUrl: https://github.com/appcircleio/appcircle-test-report-component
+repoUrl: https://github.com/appcircleio/appcircle-test-report-component.git
+commit: 02cbb2a
```

Comment:
If the PR on the component side has been merged, can you please get the latest commit from the repo for the PREP update?